### PR TITLE
Celery

### PR DESCRIPTION
**Pull request by MiklerGM**

### CODE REVIEW

File: project/api/management/commands/clean_stvs.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 from django.contrib.gis.db.models.functions import MakeValid
 
 from api.models import TerritorialEntity, SpacetimeVolume
-from api.views.stv_view import find_difference, calculate_area
+from api.helpers.geometry import find_difference, calculate_area
```
Comment (whirish) [inline project/api/management/commands/clean_stvs.py:28]: Much cleaner to have these in a separate module :+1:

**Comment by MiklerGM** [comment]
> LGTM, our next step is to develop a new endpoint to check task status?

There is one already

https://github.com/chronhq/backend/blob/b9f08441f3ae7c551c07098f32f3b432b99ff2e5/project/api/views/endpoints/stv_tasks.py#L27-L39

**Comment by MiklerGM** [comment]
bors try

**Pull request merged**